blazor/combobox/style.md

```diff
@@ -136,6 +136,34 @@ Use the following CSS to customize the appearance of popup element.
 }
 ```
 
+## Readonly-mode
+
+Specify the boolean value to [Readonly](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Readonly), whether the DropDownList allows the user to change the value.
+
+If you are setting Readonly property, then you could not interact the component. You can set initial value to the component alone.  
```

Comment:
If you set the Readonly property, you cannot interact with the component. Set the initial value to the component alone.